ksql-engine/src/main/java/io/confluent/ksql/QueryEngine.java

```diff
@@ -175,6 +187,29 @@ DDLCommandResult handleDdlStatement(
     return ksqlEngine.getDDLCommandExec().execute(command);
   }
 
+  private DDLCommand generateDDLCommand(
```

Comment:
This method shouldn't be here. I just removed it in a previous PR

Reply:
Yes, removed it again.